packages/vaadin-overlay/test/restore-focus.test.js

```diff
@@ -83,4 +83,30 @@ describe('restore focus', () => {
     await close(overlay);
     expect(overlay._getActiveElement()).to.equal(focusable);
   });
+
+  describe('restoreFocusNode', () => {
+    beforeEach(() => {
+      overlay.restoreFocusNode = focusInput;
+    });
+
+    it('should not restore focus on close by default (restore-focus-on-close=false)', async () => {
+      overlay.restoreFocusOnClose = false;
+      focusInput.focus();
+      await open(overlay);
+      // emulate focus leaving the input
+      focusInput.blur();
+      document.body.focus();
```

Comment:
It seems that focus is already on the body element after `blur()` so there is no need for `document.body.focus()`, isn't it?

Reply:
Possibly, the test is identical to the other tests in the suite with the same description.

Reply:
OK, let's not touch it then. :)